jetty-core/jetty-session/src/main/java/org/eclipse/jetty/session/SessionManager.java

```diff
@@ -26,7 +26,7 @@
  * Non-servlet spec specific contract implemented by all SessionHandlers.
  *
  */
-public interface SessionManager extends LifeCycle, SessionConfig
+public interface SessionManager extends LifeCycle, SessionConfig, Session.LifeCycleListener, Session.ValueListener
```

Comment:
The `SessionManager` is not a `Session.LifeCycleListener` nor a `Session.ValueListener`, it _calls_ those listeners.

Reply:
hhhhmmmm but it keeps the signature the same.... it made doing all the renames easy :)
But yeah, it is not a listener and would break if set as an attribute to a context or session.